source/core/src/main/com/csse3200/game/components/player/KeyboardPlayerInputComponent.java

```diff
@@ -86,6 +88,16 @@ public boolean keyDown(int keycode) {
         case Keys.NUM_9:
           triggerHotKeySelection(keycode);
           return true;
+        case Keys.C :
```

Comment:
I don't think the player should be able to trigger the different screens with keypresses.